css/responsive/desktop.styl

```diff
@@ -94,4 +94,23 @@ $margin-large = ($gutter * 5)
 // BASIC
 
 #main-content
-  padding-top: 58px + $margin-tiny
\ No newline at end of file
+  padding-top: 58px + $margin-tiny
+
+// ******************************************************
+
+// HEADER
+
+#header-menu
+  padding-top: 0
+  padding-bottom: 0
+  font-size: $font-size-h3
+  display: initial
+  li
+    display: inline-block
```

Comment:
because i switch the `:after` to a `:before` on desktop so that there isnt a comma before the last element, which is mobile-only. and with the `:before` i need to float left otherwise there is whitespace before the elem by default. already troubleshot this